### ISSUE TITLE
Ability to delete account or change personal details

### ISSUE THREAD

**Issue by lizheard1** [PlasticPatrol/PlasticPatrolWeb#126]
Do we have this functionality? E.g. if someone has a new email address and wants to set up a new account instead of the old one?

**Comment by munroTom** [comment]
We don't have this functionality in the app at the moment, but we could do it manually for users

It feels like the solution here would be to allow users to change their email instead though as I imagine they wouldn't want to lose their stats?

**Comment by lizheard1** [comment]
Yes - I think allowing users to change their email address is best, and keep all stats linked (so they don't lose them). We don't want users to delete their accounts either really so the more difficult it is for them the better
